README.md

```diff
@@ -1 +1,75 @@
-# todo-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To do list
+
+The purpose of this project is to create a to do list using webpack.
+
+## Project specifications
+
+    Set up a new project with webpack that is based on the webpack exercise you have already completed.
+    Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
+    Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
+        description [string].
+        completed [bool].
+        index: [number].
+    Write a function to iterate over the tasks array and populate an HTML list item element for each task.
+    On page load render the dynamically created list of tasks in the dedicated placeholder. The list should apear in order of the index values for each task.
+    Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
+
+## Built With
+
+- HTML
+- Javascript
+- Technologies used such as: Linters (Lighthouse, Webhint, Stylelint), Extensions (Live server, Prettier), Git, bootstrap, among others.
+  -Webpack
+
+## Authors
+
+👤 **Mario Alberto Rodriguez Cota**
+
+- GitHub: [@mariordgez](https://github.com/mariordgez)
+- Twitter: [@MarioRo75396624](https://twitter.com/MarioRo75396624)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mario-alberto-rodriguez-cota-a2860a205)
+
+## Cloning repo for non programmers
+
+1. On GitHub, navigate to the main page of the repository.
+
```

Comment:
- [ ]  In this getting started part there are some instructions that you have missed for a user to use your project. such as `npm install`, and `npm start`. And please try to follow [this](https://github.com/microverseinc/readme-template) design template.

Reply:
This issue is well resolved! ✔️